jenkins/ci.suse.de/templates/cloud-crowbar-pipeline-template.yaml

```diff
@@ -84,6 +84,15 @@
             Enable SLES/Cloud test update repos (the Cloud test update repos will
             be enabled only when cloudsource is GM based)
 
+      - bool:
+          name: update_after_deploy
+          default: '{update_after_deploy|false}'
+          description: >-
+            Run ardana update after the cloud is deployed. If true the MU's will
+            be applied only after the cloud is deployed.
```

Comment:
the sentence is really difficult to read for me. isn't this two times saying the same thing? or how are those two sentences saying something different?

Reply:
Copy-pasted. It is saying the same thing. I'll remove duplicated statement.